src/command_modules/azure-cli-interactive/tests/test_completion.py

```diff
@@ -137,8 +136,7 @@ def init4(self):
             "create --funtimes": "There is no work life balance, it's just your life"
         }
         same_param_doubles = {
-            "-f": "--funtimes",
-            "--funtimes": '-f'
+            "create": {"-f": "--funtimes", "--funtimes": '-f'}
```

Comment:
So if I had a command with --foo/-f and --doo/-d, would my dictionary look like:
```
{
  '-f': '--foo',
  '--foo': '-f',
  '-d': '--doo',
  '--doo': '-d'
}
```
?  And then the check_doubles basically removes the alias if it's matching pair is already in the command string?  Will this work if there are more than two aliases?  We have cases where there are two supported long-options (one to be deprecated in the future).  If those then have a short option, then does this still work or does it break down?